### PR TITLE
Add UW Hackweeks cluster + hub

### PR DESCRIPTION
**Pull request by yuvipanda**
Already deployed!

Ref https://github.com/2i2c-org/infrastructure/issues/962

### CODE REVIEW

**Comment by yuvipanda** [comment]
This probably took me about 90 minutes following the instructions, but almost all of that was just waiting.

File: eksctl/uwhackweeks.jsonnet [hunk 1]
```diff
@@ -0,0 +1,130 @@
+// Exports an eksctl config file for carbonplan cluster
```
[the remark this answers is not in the document]
Comment (yuvipanda) [reply eksctl/uwhackweeks.jsonnet]: @choldgraf ah yep. fixed.

**Comment by yuvipanda** [comment]
Looks like linkcheck is failing for https://github.com/organizations/2i2c-org/settings/applications/new - but that's actually the correct URL! You'll need to be a member of the 2i2c-org github org to see it, so makes sense linkcheck can't.

**Comment by yuvipanda** [comment]
@GeorgianaElena ah, great! I deployed grafana and added token.

File: config/clusters/uwhackweeks.cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                  url: https://icesat-2hackweek.github.io
+                designed_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                operated_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                funded_by:
+                  name:
+                  url: https://www.openscapes.org/
```
Comment (damianavila) [inline config/clusters/uwhackweeks.cluster.yaml:70]: Funded by openscapes? I guess it is a leftover from copy-pasting the yaml file? cc @yuvipanda

**Pull request merged**